bears/rest/RSTcheckBear.py

```diff
@@ -1,8 +1,20 @@
+import re
+
 from coalib.bearlib.abstractions.Linter import linter
 from dependency_management.requirements.PipRequirement import PipRequirement
 from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
 
 
+def _ignore_unknown(errors):
```

Comment:
@Makman2 why do we need such a function anyway? I don't understand the intention behind this function and on top of that, the regex it checks for is very weird. 
Do we even use `unittest.assertRegex`kind things in coala-bears?
Also, I might need your help with tests later.